package.json

```diff
@@ -53,34 +71,53 @@
     "karma-tap-pretty-reporter": "4.2.0",
     "karma-webpack": "git://github.com/ryanclark/karma-webpack#eec6dfc",
     "kw-doc": "2.0.0",
-    "kw-web-suite": "11.0.0",
+    "moment": "2.29.1",
     "node-pre-gyp": "0.16.0",
+    "parallel-webpack": "2.6.0",
```

Comment:
parallel-webpack is used in kw-doc. I'm not sure we need it here?

Reply:
I'll need to look into this further. kw-doc should bring it along, but running `npm run doc` complained about missing it.

Reply:
ok. kw-doc may depend on kw-web-suite which may introduce that issue.
We could migrate some of those deps in kw-doc to be proper rather than depend again on kw-web-suite

Reply:
Yep, kw-doc depends on kw-web-suite. I'll go update that first.